File: easy-solution/galaxy4276/27.java
import java.util.Arrays;

class Solution {
  public int removeElement(int[] nums, int val) {
    int i = 0;
    for (int j = 0; j < nums.length; j++) {
      if (nums[j] != val) nums[i++] = nums[j];
    }
    return i;
  }

}

public class Application {
  public static void main(String[] args) {
    Solution s = new Solution();
    int[] nums = { 3, 2, 2, 3 };
    int result = s.removeElement(nums, 3);
    Arrays.stream(nums).forEach(System.out::print);
    System.out.println(result);
  }
}

File: easy-solution/Antidote21/27.c


int removeElement(int* nums, int numsSize, int val){
    int i,index=0;
    for(i=0;i<numsSize;i++){
        if(nums[i]!=val){
            nums[index++] = nums[i];
        }
    }
    return index; 
}
   
